tests/unit/test_task_events_mgr_2.py

```diff
@@ -0,0 +1,105 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from unittest.mock import Mock
+import pytest
+
+
+from cylc.flow.task_events_mgr import TaskEventsManager
+
+
+@pytest.mark.parametrize(
+    "broadcast, remote, platforms, expected",
```

Comment:
Nothing wrong here, but my OCD makes me keeping looking at the parameter names are out of order here and in the function declaration🤣

Reply:
Good spotting. I had reordered the parameters at one point but forgot to do the same for the function args. It doesn't seem to matter, but I fixed it anyway.